CMakeLists.txt

```
@@ -108,7 +108,7 @@ option(BUILD_END_TO_END_TESTS "Build end to end tests" ON)
 option(COVERAGE "Enable coverage mapping" OFF)
 option(SHUFFLE_SUITE "Shuffle end to end test suite" OFF)
 option(LONG_TESTS "Enable long end-to-end tests" OFF)
-option(KV_STATE_RB "Enable RBMap as underlying KV state implementation" OFF)
+option(KV_STATE_RB "Enable RBMap as underlying KV state implementation" ON)
```

```suggestion
option(KV_STATE_RB "Enable RBMap as underlying KV state implementation" OFF)
```
Probably want to revert this before merging.